ReactWindows/ReactNative.Net46/UIManager/BaseViewManager.cs

```diff
@@ -140,6 +143,24 @@ public void SetTooltip(TFrameworkElement view, string tooltip)
             ToolTipService.SetToolTip(view, tooltip);
         }
 
+        /// <summary>
+        /// Detects the presence of a mouse view handler for the view.
+        /// </summary>
+        /// <param name="view">The view instance.</param>
+        /// <param name="handlerPresent">true if a mouse move handler is present.</param>
+        [ReactProp("onMouseMove")]
+        public void SetOnMouseMove(TFrameworkElement view, bool handlerPresent)
+        {
+            if (handlerPresent)
+            {
+                _viewsWithMouseMoveHandlers.Add(view);
+            }
```

Comment:
Why can't you conditionally subscribe to the event when this property is set so we aren't always firing a bunch of mouse move handlers?

Reply:
ok, will do, I didn't think it would be a big deal perf wise.

---
In reply to: [183067285](https://github.com/Microsoft/react-native-windows/pull/1787#discussion_r183067285) [](ancestors = 183067285)